frontend/src/features/stops/stopsSlice.ts

```diff
@@ -0,0 +1,38 @@
+import apiSlice from "../../app/apiSlice";
+import { type Coordinate } from "../location/locationSlice";
+
+/**
+ * A list of stops, as defined by the backend.
+ */
+export type Stops = Stop[];
+
+/**
+ * A Stop, as defined by the backend.
+ */
+export interface Stop extends Coordinate {
+  id: number;
+  name: string;
+  routeIds: number[];
+  isActive: boolean;
+}
+
+// --- API Definition ---
+
+/**
+ * This slice extends the existing API slice with the stop information routes.
+ * There should be no state to reason about here, it's just configuration.
+ */
+const stopsApiSlice = apiSlice.injectEndpoints({
```

Comment:
can you make a diagram at some point explaining how and when the frontend uses each api endpoint and the structure of the redux stuff? I think that would be helpful and good documentation to have

Reply:
I can try to, not sure what tool to use.